### PR TITLE
no auto close alert

### PR DESCRIPTION
**Pull request by doroncy**
added a flag to the alert config to disable the autohide of the alert

### CODE REVIEW

File: projects/ui-framework/src/lib/popups/alert/alert-service/alert.service.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export class AlertService {
   }
 
   public showAlert(config: AlertConfig): ComponentRef<AlertComponent> {
+    console.log('config', config);
```
Comment (galloween) [inline projects/ui-framework/src/lib/popups/alert/alert-service/alert.service.ts:30]: isForgotToRemoveConsoleLog === true ? removeConsoleLog() : sleep()

**Review by galloween — APPROVED** [review]
isApproved = true

File: projects/ui-framework/src/lib/popups/alert/alert-service/alert.service.ts (hunk 1 above)
Comment (doroncy) [reply projects/ui-framework/src/lib/popups/alert/alert-service/alert.service.ts:30]: hehe

Comment (doroncy) [reply projects/ui-framework/src/lib/popups/alert/alert-service/alert.service.ts:30]: we can rename to autoClose: boolean as well, what you think ?

**Pull request merged**